src/server/game/Battlefield/Battlefield.cpp

```diff
@@ -1076,9 +1076,9 @@ bool BfCapturePoint::Update(uint32 diff)
 
 void BfCapturePoint::SendUpdateWorldState(uint32 field, uint32 value)
 {
-    for (uint8 team = 0; team < PVP_TEAMS_COUNT; ++team)
-        for (GuidSet::iterator itr = m_activePlayers[team].begin(); itr != m_activePlayers[team].end(); ++itr)  // send to all players present in the area
-            if (Player* player = ObjectAccessor::FindPlayer(*itr))
+    for (GuidSet& activePlayers : m_activePlayers)
```

Comment:
meh, looking back at all transformations like this one I'm not sure that this brings any improvement... you now have to give `m_activePlayers[team]` a name (TYPE activePlayers : m_activePlayers tells nothing to the reader)